src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -340,6 +345,13 @@ def _merge(self, dict_1, dict_2, identifier):
     def return_response(self):
         return self.apiResponse
 
+    @staticmethod
+    def to_camel_case(s):
```

Comment:
We could declare this as `def to_camel_case(s : str)`. Also, `s` is not descriptive. Something generic like `text` also works in this case.